components/kanban/KanbanEditor.tsx

```
@@ -0,0 +1,570 @@
+            {column.cards.map((card) => (
+              <div key={card.id}>
+                {editingCard?.columnId === column.id && editingCard?.cardId === card.id ? (
+                  <div className="add-card-form">
+                    <input
+                      type="text"
+                      className="form-input"
+                      value={editCardTitle}
+                      onChange={(e) => setEditCardTitle(e.target.value)}
+                      placeholder="Card title"
+                      autoFocus
+                    />
```

Biome (2.1.2)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)